### ISSUE TITLE
[Suggestion]: Add an item/block of the Wither Storm's head as a trophy + automation of "withering" materials with trophy if Create is installed

### ISSUE THREAD

**Issue by pinkygrocket** [nonamecrackers2/crackers-wither-storm-mod#2243]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Twilight Forest and some addon mods for it (such as Umbral Skies) add boss heads as a trophy that can be displayed or worn on a player's head. While the Withered Nether Star and associated items are already trophies, something dedicated to _pure_ showing off could fit well into Minecraft's brand of silliness.

However, I also have the concept for the head trophy to serve a practical function: if Create is installed, the Wither Storm head animates when placed in front of a fan, withering items and blocks placed in front of it. This is similar to and based on how the Ender Dragon's head "enderizes" materials in Create: Halitosis and Create: Dreams and Desires. While the Withered Beacon does exist for item crafting, a fan setup could allow for easier automation with Create and similar mods.

**Comment by juanelgamer456** [comment]
yeah that's it we have to add not suggest this as part of the guidelines

**Comment by pinkygrocket** [comment]
?? Not sure what you're on about. I've used "add" before in my suggestions, and this doesn't violate any of the existing guidelines Crackers and his team have set out. I've just clarified what I'm suggesting for by changing the second part of my title.
